forms-html/Bawan-shna/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>
+    exercise
+</title>
+</head>
+
+
+<form>
+<label for="email">Enter your email:</label><br>
+<input type="email" id="email" name="email" placeholder="shnar.majeed@gmail.com" required><br>
+
+<label for="username">Username:</label><br>
+<input type="text" id="username" name="username" placeholder="bawan"><br>
+
+<label for="password">password:</label><br>
+<input type="password" id="password" name="password" placeholder="566554"><br>
+
+<label for="quantity">Quantity(between 1 and 5):</label>
+<input type="number" id="quantity" min="1" max="5" name="quantity number"><br>
+
+<label for="range">Volume(between 0 and 50):</label>
+<input type="range" id="range" name="range" min="0" max="50" ><br>
+
+
+
+<h4>radio</h4>
+
+
+<input type="radio" id="male" name="gender"  value="male" checked >
+<label for="male">Male</label><br>
+
+<input type="radio" id="female" name="gender" value="Female" >
+<label for="female">Female</label><br>
+
+
+<input type="radio" id="other" name="gender"  value="otther">
+<label for="other">Other</label><br>
+
+<h4>checkbox</h4>
+<input type="checkbox" id="bike" name="hoby1" value="I-have-a-bike" checked >
+<label for="bike">I have abike</label><br>
+
+<input type="checkbox" id="car" name="hoby2" value="I-have-a-car" >
+<label for="car">I have a car</label><br>
+
+<input type="checkbox" id="boat" name="hoby3" value="I-hav-a-boat">
+<label for="boat">I have a boat</label><br>
+
+
+
+
+<label name="button"><h4>button</h4></label><br>
+<input type="button" value="Click Me"><br>
+<label><h4>Select your favorite color: </h4></label>
+<input type="color" name="color">
+<label name ="entering date"><h4>Enter date before 1980-01-01:</h4></label><br>
+<input type="date" placeholder="12/11/1979"> <br>
+<label ><h4>date and time</h4> </label><br>
+<label><h4> (date and time):</h4></label>
+<input type="datetime">
+<br>
+<input type="submit"><br>
+<label><h4>reset from button</h4></label> <br>
+<input type="reset" name="reset">
+
+
+
+
+
+
+</form>
+
+
+<body>
+
```

Comment:
All you code that will be rendered on the screen should be inside the body tag

In your case the whole **Form** should be in the body tag